config/README.md

```diff
@@ -61,11 +61,9 @@ if err := cfg.Get("stuff.server").Populate(target); err != nil {
   // fail, we didn't find it.
 }
 
-fmt.Printf("Port is: %v\n", target.Port)
+fmt.Printf("Port is: %v", target.Port) // "Port is 8081"
```

Comment:
You can also do fmt.Println("Port is:", target.Port) or if you want to keep the format, please use %q.